api/src/auth/utils.py

```diff
@@ -20,3 +19,18 @@ def wrapper(*args, **kwargs):
             return fn(*args, **kwargs)
 
     return wrapper
+
+def authorize(roles):
+    def authorize_wrapper(fn):
+        @wraps(fn)
+        def wrapper(*args, **kwargs):
+            verify_jwt_in_request()
+            claims = get_jwt_claims()
+            for role in roles:
+                if role in claims['roles']:
+                    return fn(*args, **kwargs)
+            return "You do not have access to this page, please contact your system administrator if this is a mistake.", 403
```

Comment:
Will this message eventually be displayed for the user? What about localization?